modules/infra/versions.tf

```diff
@@ -28,4 +28,19 @@ provider "aws" {
   ignore_tags {
     keys = var.ignore_tags
   }
+
+  use_fips_endpoint = var.use_fips_endpoints
+}
+
+provider "aws" {
+  region = strcontains(var.region, "us-gov") ? "us-gov-east-1" : "us-east-1"
+  alias  = "us-east-1"
+  default_tags {
+    tags = var.tags
+  }
+  ignore_tags {
+    keys = var.ignore_tags
+  }
+
+  use_fips_endpoint = var.use_fips_endpoints
```

Comment:
why doesn't the above also have use_fips_endpoint?

Reply:
Not sure what you mean, both the default aws provider and the `"us-east-1"` alias have 
```
use_fips_endpoint = var.use_fips_endpoints
```